Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "portable-atomic"
-version = "1.7.0" #publish:version
+version = "1.7.1" #publish:version
```

Comment:
Please revert any version change. It is auto-handled at the time of release.

Also, since this is a new interface addition, the minor version should be increased, not the patch version (1.8.0 instead of 1.7.1).